### ISSUE TITLE
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'review_entity_summary.store_id'

### ISSUE THREAD

**Issue by coresh** [Smile-SA/magento2-module-elasticsuite-rating#7]
```$ bin/magento index:reindex  -vvv```
                                                                                                                                                                                                                        
```Design Config Grid index has been rebuilt successfully in 00:00:00
Customer Grid index has been rebuilt successfully in 00:00:00
Category Products index has been rebuilt successfully in 00:00:00
Product Categories index has been rebuilt successfully in 00:00:00
Product Price index has been rebuilt successfully in 00:00:00
Product EAV index has been rebuilt successfully in 00:00:00
Stock index has been rebuilt successfully in 00:00:00
Catalog Search indexer process unknown error:
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'review_entity_summary.store_id' in 'where clause', query was: SELECT `mgvm_review_entity_summary`.`entity_pk_value` AS `product_id`, avg(rating_summary) AS `ratings_summary` FROM `mgvm_review_entity_summary` WHERE (review_entity_summary.store_id = 1) AND (review_entity_summary.entity_pk_value IN(12, 23, 27, 33, 39, 50, 58, 68, 74, 78, 82, 104, 111, 127, 130, 134, 138, 147, 159, 171, 178, 182, 187, 195, 205, 217, 219, 221, 227, 230, 233, 240, 246, 255, 258, 260, 301, 306, 310, 324, 326, 347, 353, 357, 362, 367, 376, 380, 387, 392, 397, 400, 403, 407, 412, 422, 428, 436, 441, 446, 452, 471, 473, 475, 482, 486, 490, 492, 496, 500, 504, 506, 508, 511, 517, 523, 528, 533, 538, 543, 550, 552, 559, 563, 569, 575, 588, 600, 607, 630, 652, 662, 672, 674, 681, 687, 695, 701, 703, 709, 719, 727, 735, 743, 751, 757, 765, 773, 775, 777, 781, 783, 787, 793, 799, 817, 819, 829, 837, 839, 855, 857, 861, 864, 867, 878, 883, 886, 893, 899, 903, 914, 947, 952, 956, 960, 965, 970, 975, 980, 983, 988, 997, 1007, 1015, 1019, 1026, 1038, 1047, 1051, 1055, 1057, 1099, 1107, 1116, 1125, 1142, 1147, 1156, 1158, 1160, 1164, 1170, 1174, 1177, 1181, 1203, 1215, 1229, 1235, 1245, 1255, 1270, 1274, 1279, 1291, 1299, 1312, 1321, 1335, 1349, 1358, 1365, 1372, 1377, 1382, 1387, 1393, 1398, 1403, 1411, 1419, 1422, 1427, 1429, 1431, 1433, 1443, 1446, 1454, 1465, 1470, 1474, 1476, 1478, 1481, 1483, 1489, 1492, 1498, 1500, 1502, 1504, 1508, 1514, 1520, 1524, 1529, 1534, 1537, 1540, 1542, 1544, 1549, 1554, 1559, 1563, 1567, 1569, 1579, 1588, 1595, 1597, 1599, 1601, 1605, 1623, 1630, 1634, 1641, 1642, 1645, 1648, 1651, 1659, 1663, 1665, 1667, 1673, 1679, 1681, 1683, 1685, 1692, 1715, 1729, 1734, 1737, 1744, 1749, 1753, 1755, 1757, 1759, 1763, 1766, 1769, 1771, 1773, 1782, 1785, 1790, 1794, 1799, 1804, 1809, 1811, 1813, 1818, 1821, 1823, 1827, 1841, 1844, 1850, 1853, 1860, 1871, 1877, 1883, 1885, 1890, 1896, 1901, 1905, 1917, 1924, 1931, 1937, 1944, 1946, 1955, 1960, 1965, 1971, 1976, 1981, 1985, 1988, 1992, 1995, 1998, 2003, 2018, 2024, 2030, 2034, 2045, 2049, 2055, 2062, 2068, 2071, 2076, 2080, 2086, 2090, 2094, 2098, 2100, 2103, 2112, 2117, 2125, 2134, 2142, 2146, 2150, 2153, 2156, 2163, 2167, 2172, 2175, 2180, 2186, 2189, 2193, 2195, 2197, 2203, 2209, 2213, 2219, 2225, 2228, 2231, 2233, 2237, 2239, 2241, 2243, 2250, 2253, 2256, 2261, 2268, 2277, 2282, 2284, 2286, 2292, 2294, 2296, 2298, 2304, 2310, 2313, 2315, 2317, 2319, 2330, 2341, 2348, 2353, 2355, 2357, 2359, 2363, 2365, 2368, 2374, 2380, 2386, 2392, 2398, 2400, 2402, 2408, 2410, 2414, 2420, 2426, 2432, 2438, 2444, 2450, 2452, 2453, 2464, 2470, 2476, 2483, 2491, 2497, 2503, 2505, 2511, 2515, 2518, 2521, 2525, 2531, 2536, 2541, 2543, 2548, 2555, 2558, 2560, 2562, 2564, 2566, 2568, 2639)) GROUP BY `entity_pk_value`
Catalog Rule Product index has been rebuilt successfully in 00:00:00
Catalog Product Rule index has been rebuilt successfully in 00:00:00
ElasticSuite Category Indexing index has been rebuilt successfully in 00:00:00
Elasticsuite Cms Page Indexing index has been rebuilt successfully in 00:00:00
ElasticSuite Thesaurus Indexing index has been rebuilt successfully in 00:00:00```

```
$ bin/magento --version                                                                                                                                                                                                                                           
Magento CLI version 2.2.3
```

**Comment by romainruaud** [comment]
Seems to be an issue with table prefix. I see that you are using a prefix.

Will try to provide a fix for this one soon.

**Comment by romainruaud** [comment]
I fixed it on 2.1 branch and published a new minor version.

Please update to this new minor version to resolve your issue.

feel free to keep me in touch if the issue is still occuring.
